bundler.go

```diff
@@ -375,17 +375,18 @@ func (b *Bundler) bundle(e ConfigurationEnvironment) (err error) {
 	astilog.Debugf("Building for os %s and arch %s", e.OS, e.Arch)
 	var binaryPath = filepath.Join(environmentPath, "binary")
 	var cmd = exec.Command(b.pathGoBinary, "build", "-ldflags", b.ldflags.String(), "-o", binaryPath, b.pathBuild)
-	cmd.Env = []string{
-		"GOARCH=" + e.Arch,
-		"GOOS=" + e.OS,
-		"GOCACHE=" + os.Getenv("GOCACHE"),
-		"GOFLAGS=" + os.Getenv("GOFLAGS"),
-		"GOPATH=" + gp,
-		"GOROOT=" + os.Getenv("GOROOT"),
-		"PATH=" + os.Getenv("PATH"),
-		"TEMP=" + os.Getenv("TEMP"),
-		"TAGS=" + os.Getenv("TAGS"),
-	}
+	cmd.Env = os.Environ()
+	cmd.Env = append(cmd.Env,
```

Comment:
Is there a reason why you still append `GOCACHE`, `GOFLAGS`, `GOROOT`, `PATH`, `TEMP` and `TAGS`?

From my understanding, `os.Environ()` is already taking care of that, isn't it?